### PR TITLE
Add approvers to the frontends

### PR DESCRIPTION
**Pull request by kimwnasptd**
Making a PR to update the OWNERS for the frontend code of our `crud-web-app`. It had taken me way to long to send this PR, and thus I'm adding new approvers and not going gradually via first assigning reviewers.

Both @orfeas-k and @elenzio9 have both a ton of PRs for the frontends and also reviewed a lot of the PRs. Listing some for completeness:

For @orfeas-k: https://github.com/kubeflow/kubeflow/commits?author=orfeas-k
* https://github.com/kubeflow/kubeflow/pull/6899
* https://github.com/kubeflow/kubeflow/pull/6822
* https://github.com/kubeflow/kubeflow/pull/6788
* https://github.com/kubeflow/kubeflow/pull/6862

For @elenzio9: https://github.com/kubeflow/kubeflow/commits?author=elenzio9
* https://github.com/kubeflow/kubeflow/pull/6826
* https://github.com/kubeflow/kubeflow/pull/6742
* https://github.com/kubeflow/kubeflow/pull/6743
* https://github.com/kubeflow/kubeflow/pull/6754
* https://github.com/kubeflow/kubeflow/pull/6763

Then @tasos-ale has mentored me a lot, especially in the early days. Almost all of the PRs for the frontend I would send upstream were first reviewed by him internally. So it's also way overdue to give him some acknowledgement! 
* https://github.com/kubeflow/kubeflow/pull/5710
* https://github.com/kubeflow/kubeflow/pull/5567
* https://github.com/kubeflow/kubeflow/pull/6875
* https://github.com/kubeflow/kubeflow/pull/6674
* https://github.com/kubeflow/kubeflow/pull/5513

@tasos-ale is also now working with @orfeas-k on the new CentralDashboard in Angular
* https://github.com/kubeflow/kubeflow/pull/6895
* https://github.com/kubeflow/kubeflow/pull/6918

/cc @kubeflow/wg-notebooks-leads

### CODE REVIEW

**Comment by kimwnasptd** [comment]
Added one more commit for creating an OWNERS file of the common frontend code as well, since this is also used a lot

**Comment by jbottum** [comment]
@kimwnasptd note - going forward, please make an effort to add an approver in this group from another company other than Arrikto.   thanks.

**Comment by kimwnasptd** [comment]
@jbottum I think the effort needs to be put by the contributors and not me :-) 

I completely agree that more representation is always healthier for the project and I want to promote as many folks as possible. But as always this happens gradually from folks that contribute to the code (at least for the code repos). 

In this case all the above people had more than 25 PRs reviewed/merged each, and throughout the year(s). Of course there's always the chance I miss something. If you (or anyone) feel I'm overlooking the contributions of individuals I'm more than happy to discuss and define next steps to further promote these contributors and their work!

https://github.com/kubeflow/kubeflow/graphs/contributors

**Comment by surajkota** [comment]
Supporting these additions. Great work by everyone

**Comment by elikatsis** [comment]
Since everyone is in sync with the changes I'll go on and merge this PR. Thank you for the contributions @tasos-ale @orfeas-k @elenzio9 !
 
 /lgtm
 /approve

**Comment by elikatsis** [comment]
Commands didn't work? Let me try again
/lgtm
/approve
